core/server/controllers/frontend/channel-config.js

```diff
@@ -14,13 +13,13 @@ channelConfig = function channelConfig() {
             name: 'tag',
             route: '/' + config.routeKeywords.tag + '/:slug/',
             postOptions: {
-                filter: labs.isSet('hiddenTags') ? 'tags:\'%s\'+tags.hidden:false' : 'tags:\'%s\''
+                filter: 'tags:\'%s\''
```

Comment:
This points at a deeper issue, as the config is meant to be dynamic, that's why it's in a function.. but I can deal with that separately of this PR - this solution definitely works.